azurerm/import_arm_image_test.go

```diff
@@ -0,0 +1,49 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMImage_importStandalone(t *testing.T) {
+	ri := acctest.RandInt()
+	userName := "testadmin"
+	password := "Password1234!"
+	hostName := "tftestcustomimagesrc"
+	dnsName := fmt.Sprintf("%[1]s.westcentralus.cloudapp.azure.com", hostName)
+	sshPort := "22"
```

Comment:
I think we can safely default this field within the `testGeneralizeVMImage` function?